name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/merkurprivatbank/MerkurPrivatBankPDFExtractorTest.java

```diff
@@ -160,4 +169,51 @@ public void testWertpapierVerkauf01()
                         is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(0.00))));
 
     }
+
+    @Test
+    public void testKontoauszug01()
+    {
+        MerkurPrivatBankPDFExtractor extractor = new MerkurPrivatBankPDFExtractor(new Client());
+
+        List<Exception> errors = new ArrayList<>();
+
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "Kontoauszug01.txt"), errors);
+
+        assertThat(errors, empty());
+        assertThat(countSecurities(results), is(0L));
+        assertThat(countBuySell(results), is(0L));
+        assertThat(countAccountTransactions(results), is(3L));
+        assertThat(results.size(), is(3));
+        new AssertImportActions().check(results, CurrencyUnit.EUR);
+
+        assertThat(results, hasItem(deposit(hasDate("2024-01-19"), hasAmount("EUR", 900.00), //
+                        hasSource("Kontoauszug01.txt"))));
```

Comment:
To keep the tests complete, please add "hasNote" to your tests.

Thx 👍🏻

Reply:
OK, added it.